### DISCUSSION TITLE
Move tag bundles to the bottom of the dialog

### DISCUSSION THREAD

**Discussion by martienne17** [AprilSylph/XKit-Rewritten#1244] [Suggestion Box]
Just a small tweak, but I have created enough tag bundles now that every time I go to reblog something I have to scroll down 2 pages of links in order to tag that particular post. Could you possibly move the tag box to the top of this dialog?

**Comment by marcustyphoon** [comment]
If it wouldn't be too much trouble, could you post a screenshot of what you're referring to here?

**Comment by martienne17** [comment]
![image](https://github.com/AprilSylph/XKit-Rewritten/assets/142548371/1e686c5f-589d-4ae2-b022-d3eedb7a50cf)

**Reply by AprilSylph** [reply]
Are you saying you have to scroll all your Quick Tags bundles in order to get to the `Tags (comma separated)` field?

**Reply by martienne17** [reply]
I don't _have to_ but it ends up happening every time anyway lol.

And actually I just noticed that if I'm trying to reblog the last post on a page I actually _do_ have to.
